examples/biosignal-erp/plot_embedding_MEG.py

```diff
@@ -1,23 +1,23 @@
 """
-============================================
-Embedding ERP MEG data in 2D Euclidean space
-============================================
+===============================================
+Comparison of embeddings of covariance matrices
+===============================================
 
-Riemannian embeddings via Laplacian Eigenmaps (LE) and Locally Linear
-Embedding (LLE) of a set of ERP data. Embedding via Laplacian Eigenmaps is
-referred to as Spectral Embedding (SE).
+Comparison of several embeddings of a set of ERP covariance matrices extracted
+on MEG data: SE and LLE.
```

Comment:
Rather than adding a new example, I would prefer that we complete this example:
```suggestion
on MEG data: SE, LLE and t-SNE.
```